.gitignore

```diff
@@ -1 +1,2 @@
-﻿.idea/*
\ No newline at end of file
+﻿.idea/*
+.history
```

Comment:
Why was this updated?

Reply:
I using local history module for the Visual Studio Code, which saving change history locally into `.history` folder. To prevent adding these changes to the repository I add this line.

Reply:
@squallstar I don't see this change in any of the other similar PRs. Are you happy with this?

Reply:
@tonytlwu yep that's fine with me, I have that plugin too installed.